README.md

```diff
@@ -53,6 +53,74 @@ loader.show(options); // options is optional
 loader.hide();
 ```
 
+##  Angular 2 Example
+
+#### main.ts
+```js
+import { LoadingIndicator } from "nativescript-loading-indicator"; // import Loading inidicator here
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  bootstrap: [
+    AppComponent
+  ],
+  imports: [
+    HttpModule,
+    NativeScriptModule
+  ],
+  providers: [
+    LoadingIndicator // Add "LoadingIndicator" on Angular 2 providers
```

Comment:
Hi @NathanWalker in my application i'm using nativeScript + angular and this code works fine, please test this snippet on tour application